config/ecs/container-insights/otel-agent-default-config.yaml

```diff
@@ -0,0 +1,46 @@
+extensions:
+  health_check:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:55680
```

Comment:
Maybe we can mention the protocol name only without the `endpoint`. If it uses the default endpoint we can avoid adding it. See the official documentation- https://github.com/open-telemetry/opentelemetry-collector/tree/master/receiver/otlpreceiver#getting-started

Reply:
Please avoid to use `localhost` in the receivers. see https://github.com/open-telemetry/opentelemetry-collector/issues/592